Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "ShakeSizeVertical", "type": "float32" },
+        { "name": "TargetOffsetRight", "type": "float32" },
+        { "name": "TargetOffsetUp", "type": "float32" },
+        { "name": "TargetOffsetFront", "type": "float32" }
+      ]
+    },
+    "ObjAutorunCollisionBaseSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" }
+      ]
+    },
+    "ObjAutorunCollisionEndSpawner": {
+      "parent": "ObjAutorunCollisionBaseSpawner",
+      "fields": [
+      ]
+    },
+    "ObjAutorunCollisionStartSpawner": {
+      "parent": "ObjAutorunCollisionBaseSpawner",
+      "fields": [
+        { "name": "SearchPosition", "type": "vector3" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "KeepTime", "type": "float32" },
+        { "name": "EaseTime", "type": "float32" },
+        { "name": "ToPathEaseTime", "type": "float32" },
+        { "name": "ForceToGround", "type": "float32" },
+        { "name": "SendType", "type": "ObjAutorunCollisionStartSpawner::SendType" },
+        { "name": "ForcedRun", "type": "bool" }
+      ]
+    },
+    "ObjChangeDiveModeBaseSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" }
+      ]
+    },
+    "ObjChangeDiveModeEndSpawner": {
+      "parent": "ObjChangeDiveModeBaseSpawner",
+      "fields": [
+      ]
+    },
+    "ObjChangeDiveModeStartSpawner": {
+      "parent": "ObjChangeDiveModeBaseSpawner",
+      "fields": [
+        { "name": "Range", "type": "float32" }
+      ]
+    },
+    "EventDimChange3Dto2DSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "m_PathEaseTime", "type": "float32" },
+        { "name": "TargetDirection", "type": "vector3" },
+        { "name": "m_IsChangeCamera", "type": "bool" },
+        { "name": "m_IsPadCorrect", "type": "bool" },
+        { "name": "m_IsAutoChange2DPath", "type": "bool" },
+        { "name": "m_IsEnableFromFront", "type": "bool" },
+        { "name": "m_IsEnableFromBack", "type": "bool" }
+      ]
+    },
+    "EventDimChange3DtoDashSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "TargetDirection", "type": "vector3" },
+        { "name": "IsChangeCamera", "type": "bool" },
+        { "name": "IsReverseCameraEnable", "type": "bool" },
+        { "name": "Template", "type": "EventDimChange3DtoDashSpawner::Template" },
+        { "name": "IsLimitEdge", "type": "bool" },
+        { "name": "CurveCorrectionForce", "type": "float32" },
+        { "name": "PathCorrectionForce", "type": "float32" },
+        { "name": "IsEnableFromFront", "type": "bool" },
+        { "name": "IsEnableFromBack", "type": "bool" }
+      ]
+    },
```

Comment:
Unused object. The names of the parameters have been guessed from other `EventDimChange` objects and the set data parser function found at (0x800EA4D4, NTSC-J Wii).